### ISSUE TITLE
0.7.0 never connects

### ISSUE THREAD

**Issue by EvNaverniouk** [Equals182/eqFTP#154]
Just trying to run 0.7.0, but I'm unable to connect to any of my servers (which were working fine in 0.6.x). Enabling debug I see this:

```
[node-log 7:26:39 PM] [eqFTP-ftpDomain]: Queue updated: [{"type":"folder","connectionID":4,"path":"","id":"f_1435544799884110","state":"opened"}]
NodeDebugUtils.js:117 [node-log 7:26:39 PM] [eqFTP-ftpDomain]: Connecting to this ID: 4
NodeDebugUtils.js:117 [node-log 7:26:39 PM] [eqFTP-ftpDomain]: Connection already exists for this ID: 4
NodeDebugUtils.js:117 [node-log 7:26:39 PM] [eqFTP-ftpDomain]: Connecting to this ID: 4
NodeDebugUtils.js:117 [node-log 7:26:39 PM] [eqFTP-ftpDomain]: Connection already exists for this ID: 4
NodeDebugUtils.js:117 [node-log 7:26:40 PM] [eqFTP-ftpDomain]: 745: Got Directory: /public_html/staging/
NodeDebugUtils.js:117 [node-log 7:26:40 PM] [eqFTP-ftpDomain]: Got folder: /public_html/staging/
```

However, the gear icon in the UI just keeps spinning forever, and I never see a file listing.

**Comment by Equals182** [comment]
Logs looks good, though. I'll add some debug stuff for you in couple of hours.

**Comment by Equals182** [comment]
I'm not sure you're using latest version of eqFTP. I see it becuse `Queue updated` log, for example, should have `[q.a] ` prefix. And almost every command should. Please, try deleting eqFTP entirely and install it again from `https://github.com/Equals182/eqFTP`. Also, reload Node and Brackets after that.

**Comment by Equals182** [comment]
### Hello there!

I know this issue been there for a pretty long time, but I have some good news here!
We just released eqFTP v**1.0.0-beta.0** (v1-b for short I guess) and we're completely discontinuing support of all previous versions of this plugin.
New version works great on Brackets 1.8 and it should be good on lower versions too.

So yeah, we're closing and locking this issue forever and looking forward for you to try out our new version of eqFTP.
I'm sorry if I somehow disturbed you, or you lost your interest in eqFTP or Brackets in general, we're not going to make any noise here again.

If you have any questions, please contact me via email (you can find it in [my profile](https://github.com/Equals182)) or [Telegram app](http://t.me/equals182)

Thank you for understanding 😊
